code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2743,19 +2743,29 @@
 	var/ant_damage = 0
 	/// Tells the debuff how many ants we are being covered with.
 	var/amount_left = 0
+	/// List of possible common statements to scream when eating ants
+	var/list/ant_screams = list(
```

Comment:
```suggestion
	var/static/list/ant_screams = list(
```
Normally, if you've got a long-ish list of strings or whatevers that you don't override with subtypes or mutate in some way, you can use a static list for performance.

Reply:
Never knew that, thanks

Reply:
To add to this, this is very close to the usecase for string files. Not very long tho

Reply:
I haven't been planning on making it longer myself, though I don't have the slightest clue if anyone else would want to add to it later.